README.md

```
@@ -92,3 +92,56 @@
 ## 트러블 슈팅
 
 ## CONTRIBUTORS
+
+
+
+<h3 align="center">✨ Tech Stack ✨</h3>
+<div align="center">
+
+  <!-- Frontend -->
+  <img src="https://img.shields.io/badge/TypeScript-3178C6.svg?style=for-the-badge&logo=typescript&logoColor=white"/>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)